modules/cnf-du-management-pods.adoc

```diff
@@ -0,0 +1,356 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-management-pods.adoc_{context}"]
+
+= Management pods
+
+For the purposes of achieving 2-core (4 HT CPU) deployment of SNO, the set of pods that are considered _management_ are limited to:
+
+* Core Operators
+* Day 2 Operators
+* ACM pods
+
+== Core Operators
```

Comment:
In `modules/cnf-du-required-annotations-for-workload-partitioning.adoc`, I believe the Operators that are modified by some software component were referred to as "Required {product-title} pods and namespaces..."  Instead of expecting customers to interpret "Core" to mean "required", can you use "Required Operators" or maybe "Operators in the cluster management partition"

My point is that "Core" is not providing any information payload and I suspect that a previously introduced term and concept can be substituted.